pkg/services/ngalert/store/database.go

```diff
@@ -43,6 +53,7 @@ type DBstore struct {
 	DashboardService dashboards.DashboardService
 	AccessControl    accesscontrol.AccessControl
 	Bus              bus.Bus
+	InstanceStore    AlertInstanceStore
```

Comment:
Allows using different instance store implementations in DBStore: one that works with protobuf and the current implementation.